Source/VideoView.swift

```diff
@@ -81,6 +83,17 @@ class VideoView: UIView {
     }
 
     override func observeValue(forKeyPath keyPath: String?, of object: Any?, change: [NSKeyValueChangeKey: Any]?, context: UnsafeMutableRawPointer?) {
+        
+        if keyPath == VideoView.audioSessionVolumeKeyPath {
+            do {
+                try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: [])
+            }
+            catch let error {
+                print("Failed to start playback sound: \(error.localizedDescription)")
```

Comment:
We should probably handle the errors as we are doing it in other places.

https://github.com/bakkenbaeck/Viewer/blob/master/Source/VideoView.swift#L91-L92